bundle-lite.yaml

```diff
@@ -1,28 +1,29 @@
 bundle: kubernetes
+name: kubeflow-lite
 applications:
-  admission-webhook:         { charm: cs:admission-webhook-10,        scale: 1 }
-  argo-controller:           { charm: cs:argo-controller-51,          scale: 1 }
-  dex-auth:                  { charm: cs:dex-auth-60,                 scale: 1 }
-  istio-ingressgateway:      { charm: cs:istio-ingressgateway-20,     scale: 1 }
-  istio-pilot:               { charm: cs:istio-pilot-20,              scale: 1, options: { default-gateway: "kubeflow-gateway" }  }
-  jupyter-controller:        { charm: cs:jupyter-controller-56,       scale: 1 }
-  jupyter-ui:                { charm: cs:jupyter-ui-10,               scale: 1 }
-  kfp-api:                   { charm: cs:kfp-api-12,                  scale: 1 }
-  kfp-db:                    { charm: cs:~charmed-osm/mariadb-k8s-35, scale: 1, options: { database: mlpipeline } }
-  kfp-persistence:           { charm: cs:kfp-persistence-9,           scale: 1 }
-  kfp-schedwf:               { charm: cs:kfp-schedwf-9,               scale: 1 }
-  kfp-ui:                    { charm: cs:kfp-ui-12,                   scale: 1 }
-  kfp-viewer:                { charm: cs:kfp-viewer-9,                scale: 1 }
-  kfp-viz:                   { charm: cs:kfp-viz-8,                   scale: 1 }
-  kubeflow-dashboard:        { charm: cs:kubeflow-dashboard-56,       scale: 1 }
-  kubeflow-profiles:         { charm: cs:kubeflow-profiles-52,        scale: 1 }
-  kubeflow-volumes:          { charm: cs:kubeflow-volumes-0,          scale: 1 }
-  minio:                     { charm: cs:minio-55,                    scale: 1 }
-  mlmd:                      { charm: cs:mlmd-5,                      scale: 1 }
-  oidc-gatekeeper:           { charm: cs:oidc-gatekeeper-54,          scale: 1 }
-  pytorch-operator:          { charm: cs:pytorch-operator-53,         scale: 1 }
-  seldon-controller-manager: { charm: cs:seldon-core-50,              scale: 1 }
-  tfjob-operator:            { charm: cs:tfjob-operator-1,            scale: 1 }
+  admission-webhook:             { charm: admission-webhook,       channel: edge, scale: 1 }
+  argo-controller:               { charm: argo-controller,         channel: edge, scale: 1 }
+  dex-auth:                      { charm: dex-auth,                channel: edge, scale: 1 }
+  istio-ingressgateway-operator: { charm: istio-gateway,           channel: latest/edge, scale: 1, trust: true, options: { kind: ingress } }
+  istio-pilot:                   { charm: istio-pilot,             channel: edge, scale: 1, options: { default-gateways: "kubeflow-gateway" } }
+  jupyter-controller:            { charm: jupyter-controller,      channel: edge, scale: 1 }
+  jupyter-ui:                    { charm: jupyter-ui,              channel: edge, scale: 1 }
+  kfp-api:                       { charm: kfp-api,                 channel: edge, scale: 1 }
+  kfp-db:                        { charm: cs:~charmed-osm/mariadb-k8s-35, scale: 1, options: { database: mlpipeline } }
```

Comment:
We should point to this chamr in [Charmhub](https://charmhub.io/charmed-osm-mariadb-k8s)